src/models/gaze17/repairs.md

```diff
@@ -0,0 +1,205 @@
+5. Unplug the two white fan connectors from the motherboard (highlighted cyan above).
+6. Remove the heatsink/fans from the case, being careful not to bend the heatsink pipes. It may take some pressure to break the seal of the thermal paste.
+7. Using a paper towel, remove the existing thermal paste. You may also use a small amount of rubbing alcohol if the old paste is dried or difficult to remove.
+
+![Thermal paste removal](./img/thermal-paste-removal.webp)
+
+8. After cleaning the CPU, GPU, and heatsink, apply a small line of thermal paste directly onto the CPU and GPU chips.
+
+![Thermal paste application](./img/thermal-paste-application.webp)
+
+9. Carefully replace the heatsink.
+10. Replace the silver heatsink screws, starting with #1, then #2, and so on until #6.
+11. Replace the silver fan screws.
+12. Plug the two white fan connectors back into the motherboard.
+
+## Replacing the keyboard:
+
+Keyboard replacement is simple and requires only a cross-head screwdriver.
+
+**Tools required:** Cross-head (Phillips) screwdriver  
+**Time estimate:** 5 minutes  
+**Difficulty:** Medium <span style="color:orange;">●</span>
+
+### Steps to replace the keyboard:
+
+1. Follow the steps above to [remove the bottom cover](#removing-the-bottom-cover).
+2. Locate the keyboard adhesive strip at the edge of the motherboard underneath the right fan's heatsink pipe, highlighted green below.
+
+![Keyboard adhesive strip location](./img/keyboard-adhesive-location.webp)
+
+3. Using a pair of tweezers, pull out the end of the adhesive strip.
+
+![Keyboard adhesive strip removal](./img/keyboard-adhesive-removal.webp)
+
+- Alternatively, to remove the keyboard without tweezers, you can [remove the heatsink](#replacing-the-cooling-system) so the adhesive strip is exposed.
+    - This method is not recommended unless you are also replacing the thermal paste.
+
+![Exposed keyboard adhesive strip](./img/keyboard-adhesive-exposed.webp)
+
+4. Open the lid slightly and place the machine on its side.
+5. Push a screwdriver into the keyboard push point (next to the RAM slot) until the keyboard pops out.
+
+![Keyboard push point](./img/keyboard-push-point.webp)
+
+6. Set the machine back down and raise the keyboard away from the chassis. The larger ribbon cable is for the keyboard, while the smaller ribbon cable is for the keyboard backlight.
+
+![Keyboard ribbon connectors](./img/keyboard-ribbons.webp)
+
+7. Flip the black latches upwards to free the ribbon cables.
+8. Pull the ribbon cables out of the connectors.
+9. Remove the keyboard and replace it with the new one.
+10. Carefully slide both ribbon cables into their connectors.
+11. Flip the black latches back into place to secure the ribbon cables.
+12. Place the keyboard back into position, starting with the tabs on the bottom edge.
+13. Secure the rest of the keyboard by pressing down on each of its edges.
+14. Turn the machine lid-side down again and replace the two keyboard screws.
```

Comment:
`replace the two keyboard screws` should probably be `[replace the bottom cover](#removing-the-bottom-cover.)`

Reply:
Fixed with df22a2805443e1e1abfa9f9df4d59b6aa03ff100.